scheduler/drilling-wits-depth-median/src/service.py

```diff
@@ -0,0 +1,90 @@
+import collections
+import itertools
+import statistics
+from typing import Any, Dict, List, Optional, Union
+
+from src import models
+
+
+def get_stats(data: List[dict]) -> Dict[str, Union[float, List[Optional[float]]]]:
+    # find unique keys from all data dicts
+    keys = set(itertools.chain(*data))
+
+    stats: Dict[str, Union[float, List[Optional[float]]]] = {}
+
+    for key in keys:
+        values = []
+
+        for datum in data:
```

Comment:
```suggestion
    values = [datum[key] for datum in data if key in datum]
```

I think this is identical functionality

Reply:
thanks, that's easier to read